src/assets/index.js

```diff
@@ -238,6 +238,7 @@ function validateAndDisplay(fileNotImage = false) {
         processData: false,
         success: function (data) {
             console.log("Successful reception of data!!");
```

Comment:
do it like this, and update accordingly.
success: function (analysis_report_json)

READ THE ISSUE ONCE AGAIN